ThreadGroup/Controllers/TGNavigationViewController.m

```diff
@@ -235,4 +236,11 @@ - (UIAlertController *)moreMenu {
     return _moreMenu;
 }
 
+#pragma mark - UINavigationControllerDelegate
+
+- (id<UIViewControllerAnimatedTransitioning>)navigationController:(UINavigationController *)navigationController animationControllerForOperation:(UINavigationControllerOperation)operation fromViewController:(UIViewController *)fromVC toViewController:(UIViewController *)toVC {
+    TGNavigationAnimator *animator = [TGNavigationAnimator new];
+    return animator;
+}
```

Comment:
So, `TGRootVC` is the delegate for the navigation stack, `TGNavigationController` is never set as the delegate. So you can remove all the transition code from `TGNavigationController`